modules/tinymce/src/themes/silver/main/ts/ui/context/ContextFormApi.ts

```diff
@@ -20,6 +22,7 @@ export const getFormApi = <T>(input: AlloyComponent): InlineContent.ContextFormI
         valueState.set(Representing.getValue(input));
       }
 
+      AlloyTriggers.emit(input, backSlideEvent);
```

Comment:
Since this changes how hide works I wonder if we should make this a new api function instead that is called `back` ? You might still want to hide it depending on what the action is?

Reply:
we could but I'm not entirely sure about it, because what that event does (if I understood correctly) is [set a stack](https://github.com/tinymce/tinymce/blob/main/modules/tinymce/src/themes/silver/main/ts/ui/context/ContextUi.ts#L93-L102) to the correct state, I don't think not setting that stack is the correct behavior anyway, I'll do some check to see if there is some other unexpected behavior

Reply:
I see, I didn't thought about that, I'll create a new function